src/Sharpl/Fix.cs

```diff
@@ -9,49 +10,59 @@ public static class Fix
     public static readonly byte HeaderBits = 5;
     public static readonly byte ExpBits = 4;
 
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
All methods on this type should really be inlined, but NativeAOT occasionally errs on the side of caution even on smaller methods. When inspecting disassembly this type looked like that case, which these attributes fix.